README.md

```diff
@@ -1,1214 +1,322 @@
-<h1 align="center">OpenAPI Generator</h1>
```

Comment:
@icubbon thanks for the PR. Why deleted the project's README ?

Reply:
@wing328 The `ci/circleci: node1` tests are failing with the below message. Is this a flaky test framework setup issue?
```
Execution default of goal org.springframework.boot:spring-boot-maven-plugin:3.0.0:repackage failed: Unable to load the mojo 'repackage' in the plugin 'org.springframework.boot:spring-boot-maven-plugin:3.0.0' due to an API incompatibility: org.codehaus.plexus.component.repository.exception.ComponentLookupException: org/springframework/boot/maven/RepackageMojo has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```